source/ihc_core.c

```diff
@@ -276,13 +274,16 @@ static int ihc_get_V4_bng_MAC_address(char *ipAddress, char *MACAddress, unsigne
 
     if( strlen(ipAddress) > 0 )
     {
-        memset(command, 0, IHC_MAX_STRING_LENGTH);
+        char command[IHC_MAX_STRING_LENGTH];
```

Comment:
Please, in variable declaration it's preferable to initialise them to avoid unexpected behaviour.